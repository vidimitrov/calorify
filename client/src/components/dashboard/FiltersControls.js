import styled from 'styled-components';

const FiltersControls = styled.div`
  display: ${props => (props.visible ? 'inline-block' : 'none')};
  flex-grow: 1;

  > div {
    margin: 0 10px;
  }

  @media (max-width: 420px) {
    > div {
      margin: 10px;
    }
  }
`;

export default FiltersControls;
