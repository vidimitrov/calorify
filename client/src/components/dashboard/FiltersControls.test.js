import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import FiltersControls from './FiltersControls';

it('Dashboard/FiltersControls component renders correctly', () => {
  const tree = renderer.create(<FiltersControls />).toJSON();
  expect(tree).toMatchSnapshot();
});
