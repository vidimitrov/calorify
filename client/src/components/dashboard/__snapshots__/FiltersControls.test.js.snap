// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard/FiltersControls component renders correctly 1`] = `
.c0 {
  display: none;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c0 > div {
  margin: 0 10px;
}

@media (max-width:420px) {
  .c0 > div {
    margin: 10px;
  }
}

<div
  className="c0"
/>
`;
